el-patch-template.el

```diff
@@ -0,0 +1,513 @@
+;;; el-patch-template.el --- -*- lexical-binding: t -*-
+
+
+;; Author: Al Haji-Ali <abdo.haji.ali@gmail.com>
+;; Created: 1 March 2021
+;; Homepage: http://www.randomoid.com
```

Comment:
This should probably be

```suggestion
;; Homepage: https://github.com/raxod502/el-patch
```

since I think it is for the package not hte author.